backend/main/chapters/c11_tic_tac_toe_project.py

```diff
@@ -7,6 +7,31 @@
 from main.text import ExerciseStep, Page, MessageStep, Disallowed
 
 
+def generate_board(board_type):
+    winning, size = choice([True, False]), randint(3, 10)
+    char1, char2 = choice(ascii_uppercase), choice(ascii_uppercase)
```

Comment:
I don't like this style, please just assign one per line like before.

Reply:
OK.